sections/featured-product.liquid

```diff
@@ -304,86 +304,25 @@
         <button id="ModalClose-{{ section.id }}" type="button" class="product-media-modal__toggle" aria-label="{{ 'accessibility.close' | t }}">{% render 'icon-close' %}</button>
 
         <div class="product-media-modal__content" role="document" aria-label="{{ 'products.modal.label' | t }}" tabindex="0">
```

Comment:
I noticed that it's missing the behaviour where you can click on the background/image to close the modal like on the product page. Is that something added in another PR? Or something we need to look into ?

Reply:
Fixed. Good catch :dancing-razz: